packages/manager/apps/container/src/container/legacy/server-sidebar/order/OrderPopupContent.tsx

```diff
@@ -19,16 +28,60 @@ export interface ShopSubItem {
   external?: boolean;
   tracking?: string;
 }
+
 export interface ShopItem {
   external?: boolean;
   url?: string;
-  icon: string | ComponentWithAs<'svg', IconProps>;
+  icon: any;
   label: string;
   subMenu?: Array<ShopSubItem>;
   feature?: string;
   tracking?: string;
 }
 
+const OpenLayoutContent = ({ item }: any) => {
+  const { t } = useTranslation('server-sidebar-order');
+  return (
+    <>
+      <div className="order-popup-content-icon">{item.icon}</div>
```

Comment:
**question:** Are you sure to use `block` element to display icon?
```suggestion
      <div className="order-popup-content-icon" aria-hidden="true">{item.icon}</div>
```

Reply:
👍